src/omlt/io/onnx_parser.py

```diff
@@ -464,20 +475,14 @@ def _consume_pool_nodes(self, node, next_nodes):
         in_channels = input_output_size[0]
 
         attr = _collect_attributes(node)
-        kernel_depth = attr["kernel_shape"][0]
+        kernel_depth = in_channels
         kernel_shape = attr["kernel_shape"][1:]
```

Comment:
I think kernel_shape = attr["kernel_shape"] since the first dimension is no longer the depth in maxpooling. Can you put some tests on this part to see which one is correct?